data/txHandler.go

```diff
@@ -46,6 +47,9 @@ var transactionMessagesHandled = metrics.MakeCounter(metrics.TransactionMessages
 var transactionMessagesDroppedFromBacklog = metrics.MakeCounter(metrics.TransactionMessagesDroppedFromBacklog)
 var transactionMessagesDroppedFromPool = metrics.MakeCounter(metrics.TransactionMessagesDroppedFromPool)
 
+// verifierStreamBufferSize is the number of txn that coult be accumulated before the verifier stream consumes them
```

Comment:
"number of txn that _could_ be accumulated"
or
"number of txn that _couldn't_ be accumulated"

Reply:
verifierStreamBufferSize should not be. No need to buffer here.